src/Schema/Event.ts

```diff
@@ -93,3 +105,203 @@ export interface SuccessfullyLoggedIn {
   auth_redirect: string
   service: AuthService
 }
+
+/**
+ * A user taps a grouping of artists on iOS
+ *
+ * This schema describes events sent to Segment from [[tappedArtistGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedArtistGroup"
+ *    context_module: "trendingArtistsRail"
+ *    context_screen_owner_type: Home
+ *    destination_screen_owner_type: Artist
+ *    destination_screen_owner_id: "5359794d1a1e86c3740001f7"
+ *    destination_screen_owner_slug: "anthony-hunter"
+ *    horizontal_slide_position: 1
+ *    module_height: "double"
+ *    type: "stub"
+ *  }
+ * ```
+ */
+export interface tappedArtistGroup {
```

Comment:
This is minor but interfaces here and elsewhere are [generally CapsCase](https://github.com/artsy/cohesion/blob/master/src/Events/Authentication/CreatedAccount.ts#L10) so that they can be distinguished from common variables / functions when later imported into code.